[2021-12-08 14:53:32,704] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.pre_process scheduled__2021-08-29T00:00:00+00:00 [queued]>
[2021-12-08 14:53:32,762] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: cpp_workflow.pre_process scheduled__2021-08-29T00:00:00+00:00 [queued]>
[2021-12-08 14:53:32,777] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 14:53:32,779] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2021-12-08 14:53:32,779] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 14:53:32,901] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): pre_process> on 2021-08-29 00:00:00+00:00
[2021-12-08 14:53:32,923] {standard_task_runner.py:52} INFO - Started process 361 to run task
[2021-12-08 14:53:32,990] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'cpp_workflow', 'pre_process', 'scheduled__2021-08-29T00:00:00+00:00', '--job-id', '240', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpldjhtijx', '--error-file', '/tmp/tmpi3y70f6d']
[2021-12-08 14:53:32,996] {standard_task_runner.py:77} INFO - Job 240: Subtask pre_process
[2021-12-08 14:53:33,223] {logging_mixin.py:109} INFO - Running <TaskInstance: cpp_workflow.pre_process scheduled__2021-08-29T00:00:00+00:00 [running]> on host 8edf9af2b560
[2021-12-08 14:53:33,577] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=cpp_workflow
AIRFLOW_CTX_TASK_ID=pre_process
AIRFLOW_CTX_EXECUTION_DATE=2021-08-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-29T00:00:00+00:00
[2021-12-08 14:53:58,413] {local_task_job.py:154} INFO - Task exited with return code Negsignal.SIGKILL
[2021-12-08 14:53:58,637] {taskinstance.py:1280} INFO - Marking task as FAILED. dag_id=cpp_workflow, task_id=pre_process, execution_date=20210829T000000, start_date=20211208T145332, end_date=20211208T145358
[2021-12-08 14:53:59,259] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
